.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: VPC deployment in AWS
+on:
+  push:
+    branches:
+      githubactions_vpc
```

Comment:
This branch will not exists after the merge. So it should set to main branch or any other branches instead.

Reply:
Branch name has been changed to main.